src/integration-test/java/com/commercetools/sync/integration/externalsource/categories/CategorySyncIT.java

```diff
@@ -828,7 +828,7 @@ void syncDrafts_WithDraftWithAMissingParentKey_ShouldNotSyncIt() {
     final CategorySyncStatistics syncStatistics =
         categorySync.sync(categoryDrafts).toCompletableFuture().join();
 
-    assertThat(syncStatistics).hasValues(2, 2, 0, 0);
+    assertThat(syncStatistics).hasValues(2, 1, 0, 1);
```

Comment:
This scares out me, why it should be failed? The previous IT scenario looks valid 🤔

Reply:
it failed, because one Category cannot be imported, because its parent category doesn`t  exist and doesn´t get imported in the batch. So this category is saved in a customobject  and the failed statistic is incremented.

Reply:
This is weird, is it really "failing" or something like "pending"? This seems to be connected with the "decrementation" hack...

Reply:
no its failing because it couldn`t be imported in the current sync process